src/node_util.cc

```diff
@@ -91,11 +91,10 @@ static void GetProxyDetails(const FunctionCallbackInfo<Value>& args) {
   if (!args[0]->IsProxy())
     return;
 
-  CHECK(args[1]->IsBoolean());
-
   Local<Proxy> proxy = args[0].As<Proxy>();
 
-  if (args[1]->IsTrue()) {
+  // The length check keeps this function backwards compatible.
```

Comment:
I think it would be good to name-and-shame here, otherwise the comment is going to befuddle future maintainers. Things at the binding layer don't normally have to worry about backwards compatibility.

Reply:
I changed it to a TODO statement. PTAL.